src/cosmos-copilot.WebApp/Services/SemanticKernelService.cs

```diff
@@ -122,8 +122,8 @@ public SemanticKernelService(OpenAIClient openAiClient, CosmosClient cosmosClien
 
         //Create a tokenizer for the model
         _tokenizer = Tokenizer.CreateTiktokenForModel(modelName: "gpt-4o");
-        _maxRagTokens = Int32.TryParse(maxRagTokens, out _maxRagTokens) ? _maxRagTokens: 3000;
-        _maxContextTokens = Int32.TryParse(maxContextTokens, out _maxContextTokens) ? _maxContextTokens : 1000;
```

Comment:
This should be 2500 tokens.

Reply:
I thought rag was 2500 and context was 500?